addons/website_slides/tests/test_slide_slide.py

```diff
@@ -115,3 +115,37 @@ def test_video_vimeo(self):
                     })
                     self.assertEqual('vimeo', slide.video_source_type)
                     self.assertEqual(vimeo_id, slide.vimeo_id)
+
+    def test_compute_category_completion_time(self):
```

Comment:
Could you add a small docstring explaining the purpose of the test ? Otherwise it may not be obvious why it exists at all